docs/performance.md

```diff
@@ -129,4 +131,50 @@ MiB Swap:      0.0 total,      0.0 free,      0.0 used.   2989.2 avail Mem
 
 ### AWS
 
-Coming soon.
+The indicative figures below were measured using the [CT hammer tool](/internal/hammer/) as of [commit `2940bba`](https://github.com/transparency-dev/static-ct/commit/2940bba60a49bad1b78ae8f2ded2c893b1b133ad). 
+
+#### t3a.small EC2 Instance + Aurora MySQL db.r5.large
+
+- t3a.small (2 vCPUs, 2 GB Memory)
+  - General Purpose SSD (gp3)
+    - IOPS: 3000
+    - Throughput: 125
```

Comment:
Is there a unit to go with this `Throughput` number?

Reply:
Added, thanks!